Gemfile.lock

```diff
@@ -293,6 +306,12 @@ GEM
       tilt (>= 1.1, < 3)
     sdoc (1.0.0.rc1)
       rdoc (= 5.0.0)
+    sekrets (1.9.1)
+      coerce (>= 0.0.3)
+      fattr (>= 2.2.1)
+      highline (>= 1.6.15)
+      main (>= 5.1.1)
+      map (>= 6.3.0)
```

Comment:
I was thinking the same.

I'm not sure if this is really worth it to integrate since you can do almost all of this as described in https://github.com/ahoward/sekrets/blob/master/README when you need this feature. I think the only difference is key generation (where Sekrets will ask you for key). That can be solved in Sekrets and I think it will be enough to update guides to recommend this approach for storing secret data within project.

But maybe I'm missing something.